locales/cy/furniture.json

```diff
@@ -35,5 +35,7 @@
   "yes": "to be translated",
   "no": "to be translated",
   "companyName": "to be translated",
-  "companyNumber": "to be translated"
+  "companyNumber": "to be translated",
+  "next" : "Nesaf",
```

Comment:
Should these be 'to be translated'?

Reply:
Yes, that's the Jira I was told to pick up. 
DACT-1424 - WLE - Core - Current directors page - Add Welsh language translations

it's already done under POC, just few bits were missed out.